test/compilable/cppmangle.d

```diff
@@ -686,6 +717,6 @@ version (Win64)
     static assert(TestOperators.opCast!int.mangleof      == "??BTestOperators@@QEAAHXZ");
     static assert(TestOperators.opAssign.mangleof        == "??4TestOperators@@QEAAHH@Z");
     static assert(TestOperators.opEquals.mangleof        == "??8TestOperators@@QEAA_NH@Z");
-    static assert(TestOperators.opIndex.mangleof         == "??ATestOperators@@QEAAH_K@Z");
+    static assert(TestOperators.opIndex.mangleof         == "??ATestOperators@@QEAAHH@Z");
```

Comment:
How was this passing before? This PR doesn't fix it.

Reply:
I'm not sure what you mean? All the tests pass...

Before it was `size_t`, I changed it to `int` (look above) because 32/64 mangles `size_t` differently so the posix tests which aren't split on 32/64bit needed a stable type to test.

Reply:
ok